README.md

```diff
@@ -51,27 +51,9 @@ Assumptions:
         $ npm install
 
 
-### Create Configurations
+### Configuration File
 
-The `usaspending-website` folder provides three sample `GlobalConstants` files:
-
- * `sampleGlobalConstants_dev.js`
- * `sampleGlobalConstants_prod.js`.
-
-Use these sample files to create files named `GlobalConstants_dev.js` and `GlobalConstants_prod.js` respectively.
-
-You **must** have *both* `GlobalConstants_dev.js` and `GlobalConstants_prod.js` created before building the application. 
-
-The sample files require you to provide values for:
-
-* `API` (string) is the base API URL for the server that is hosting the API.
-	* The USAspending API is located at `https://api.usaspending.gov/api/`, with the trailing slash included.
-* `DEV` (boolean) indicates if you are running the application in development mode or for production use. Enabling development mode activates certain debugging functionality at the expense of some performance.
-* `PERF_LOG` (boolean) indicates if you want to enable performance logging data in the JavaScript console. This requires `DEV` to be enabled as well.
-
-`DEV` and `PERF_LOG` should be disabled when deploying to a hosted public, staging, or production environment.
-
-**TIP!** You can use separate `GlobalConstants_dev.js` and `GlobalConstants_prod.js` files to point to different API endpoints that align with different environments.
+The `usaspending-website` folder provides a single configuration file named `GlobalConstants.js`. Here you may adjust the `API` property to use either a local api or the production api. By default, `npm run start` uses the local API url. To use the production API, simply overwrite the `API` property or rewrite the `npm` script to use `webpack.prod.config.js`.
```

Comment:
Recommend specifying the local API url (http://localhost:8000/api/) in the documentation